2024/04/an-openstreetmap-editor-for-incorporating-gtfs-data.md

```diff
@@ -0,0 +1,85 @@
+
+Enter the General Transit Feed Specification (GTFS), a widely adopted standard for managing public transportation data. I wanted to explore ways to integrate OSM with GTFS effectively.
+
+### My OpenStreetMap editor for GTFS data
+
+A simple approach for addressing this challenge would be to overlay an OSM map with stops from GTFS. While this would be relatively straightforward, it doesn't leverage OSM's biggest strength—its vibrant community. One of OSM's most significant advantages is that it dynamically incorporates user contributions: as soon as the community accesses useful data, members enhance the details and accuracy of its mapping data. Therefore, I’ve been working on truly integrating GTFS data with OSM, creating a cohesive mapping experience rather than settling for a disjointed overlay.
+
+Although the implementation is a bit trickier, I believe the better approach for handling this challenge involves merging GTFS stops with those already present in OSM and incorporating GTFS stop codes into OSM data. Hence, I've developed a single-page online editor. This tool facilitates the creation, editing, and deletion of stops within OSM. It also highlights stops based on their alignment with stops specified in a given GTFS file.
+
+Editing the routes wasn’t a priority, yet I found it necessary to match GTFS routes and trips with OSM routes and their variants. Displaying trips shows which part of a street a route uses and also which side of a street bus stops are located on.
+
+I've deployed the app on my GitHub, where you can try it out: https://kiselev-dv.github.io/osm-gtfs/.
+
+### How to use the app
+
+To start, download a GTFS data dump. They are usually published by local transit companies. As an example, I'll use data from Halifax, Nova Scotia.
+
+Open the GTFS ZIP file in the editor. The editor should parse stops from GTFS.
+
+![The OSM editing app. The map is not darkened and does not have any blue, red, or black dots. The left bar does not have a list of tags anymore.](/blog/2024/04/an-openstreetmap-editor-for-incorporating-gtfs-data/gtfs-file.webp)
+
+Next, query OSM data and select “OSM Tag with GTFS stop code.” After OSM data is loaded you will get OSM tags statistics which might help you to see if there are some popular tag combination for GTFS data in the area. OSM doesn’t have strict data model, but as a rule of thumb for that type of data `ref` or `gtfs:ref` should work. If there are multiple agencies sharing a stop and each of them publishes its own GTFS data, you can use something like `gtfs:<agency_name>:ref`. By default the editor will substitute the most popular tag containing `ref` or `gtfs`.
+
+Now you can match stops from OSM with stops from GTFS. Matches are color coded:
+
+* Blue: matching stops
+* Red: stops which are present in GTFS but not mapped in OSM — stops to add
```

Comment:
I would remove the dash and put it in parentheses, since this is a clarification

Reply:
And do the same with the next line